activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeUpdateTaskTest.java

```diff
@@ -0,0 +1,153 @@
+                                                            .build());
+        tasks = taskRuntime.tasks(Pageable.of(0,
+                                              50));
+
+        assertThat(RuntimeTestConfiguration.updatedTasks).contains(updatedTask.getId());
+        assertThat(tasks.getTotalItems()).isEqualTo(1);
+        assertThat(tasks.getContent())
+                .filteredOn("status",
+                            Task.TaskStatus.ASSIGNED)
+                .containsOnly(updatedTask);
+    }
+
+    @Test
+    @WithUserDetails(value = "garth", userDetailsServiceBeanName = "myUserDetailsService")
+    public void bCreateClaimAndUpdateStandaloneTask() {
+
+        String authenticatedUserId = securityManager.getAuthenticatedUserId();
+
+        // create
+        Task standaloneTask = taskRuntime.create(TaskPayloadBuilder.create()
+                                                         .withName("test task update")
+                                                         .withDescription("test task update description")
+                                                         .withDueDate(new Date())
+                                                         .withPriority(50)
+                                                         .build());
+
+        assertThat(RuntimeTestConfiguration.createdTasks).contains(standaloneTask.getId());
+
+        Page<Task> tasks = taskRuntime.tasks(Pageable.of(0,
+                                                         50));
+
+        assertThat(tasks.getTotalItems()).isEqualTo(2);
+        assertThat(tasks.getContent())
+                .extracting("status",
+                            "id")
+                .contains(tuple(Task.TaskStatus.CREATED,
+                                standaloneTask.getId()));
+
+        final UpdateTaskPayload updateTaskPayload = TaskPayloadBuilder.update()
+                .withTaskId(standaloneTask.getId())
+                .withName(standaloneTask.getName() + " [UPDATED]")
+                .withPriority(60)
+                .withDueDate(new Date())
+                .withDescription(standaloneTask.getDescription() + " [UPDATED]")
+                .build();
+
+        // try update
+        Throwable thrown = catchThrowable(() -> taskRuntime.update(updateTaskPayload)); // task should be claimed before be updated
+        assertThat(thrown).isInstanceOf(IllegalStateException.class).hasMessage("You cannot update a task where you are not the assignee");
+
+        // claim
+        taskRuntime.claim(TaskPayloadBuilder.claim().withTaskId(standaloneTask.getId()).build());
+
+        // update
+        final Task updatedTask = taskRuntime.update(updateTaskPayload);
+        tasks = taskRuntime.tasks(Pageable.of(0,
+                                              50));
+
+        assertThat(RuntimeTestConfiguration.updatedTasks).contains(updatedTask.getId());
+        assertThat(tasks.getTotalItems()).isEqualTo(2);
```

Comment:
Same here, I don't think you need to check the total elements.

Reply:
I removed it.